src/Packagist/WebBundle/Resources/views/package/view_package.html.twig

```diff
@@ -174,6 +174,16 @@
                             {% endif %}
                         </div>
 
+                        <div class="facts col-xs-12 col-sm-6 col-md-12">
+                            {% if version and version.funding is defined and version.funding|length > 0 %}
```

Comment:
How can `version.funding` be undefined ? Will `version` be something different than the `Version` entity ?

Note that `is defined` is **not** a null check.